partner_accreditation/models/res_partner_accreditation.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class ResPartnerAccreditation(models.Model):
+    _name = "res.partner.accreditation"
+    _description = "Configurable Accreditation for Partners"
+
+    name = fields.Char(string="Accreditation")
+    res_partner_id = fields.Many2many("res.partner")
```

Comment:
Should it be `res_partner_ids` with an `s` because it's `many2many` ? Also, often we just use `partner_ids` not specifying the `res_`.